editor_drafts/sd-cwt-example.cbor-diag

```diff
@@ -0,0 +1,76 @@
+/ cose-sign1 / 18([
+  / protected / << {
+    / alg / 1  : -35, / ES384 /
+    / typ / 16 : "application/sd+cwt",
+    / kid / 4  : 'https://issuer.example/cwt-key3'
+  } >>,
+  / unprotected / {
+    / sd_claims / 17 : [  /these are the three disclosures/
```

Comment:
Again, I would argue to encode these once, a single << >>.
In large redaction sets there will also be a performance cost to encoding each disclosed claim instead of the set all at once.

Reply:
Since disclosures happen selectively per claim, I think we need to encode each one separately.